src/App.js

```diff
@@ -34,6 +34,9 @@ function App() {
         <PassingProsComponent prop1="1" prop2="2">
           123
         </PassingProsComponent>
+        <PassingProsComponent prop1="1" prop3="2">
+          123
+        </PassingProsComponent>
```

Comment:
**Suggestion:** It seems like you are passing a new prop 'prop3' to the 'PassingProsComponent' but you have removed 'prop2'. If 'prop2' is still needed, you should include it. If not, you should remove it from the other 'PassingProsComponent' instances.
```suggestion
        <PassingProsComponent prop1="1" prop2="2" prop3="2">
          123
        </PassingProsComponent>
```